.github/workflows/publish.custom.form.field.yml

```diff
@@ -0,0 +1,62 @@
+# This workflow will run tests using node and then publish a package to GitHub Packages when a release is created
+# For more information see: https://help.github.com/actions/language-and-framework-guides/publishing-nodejs-packages
+
+name: Kf Create component Publish
```

Comment:
exisiting kf-create-component itself we can have this action la why new ?

Reply:
Renamed the github action's name, this action is for releasing 3 packages releated to form-field custom components.